ios/Capacitor/Capacitor/CAPInstanceDescriptor.h

```diff
@@ -63,15 +69,15 @@ NS_SWIFT_NAME(InstanceDescriptor)
  */
 @property (nonatomic, retain, nonnull) NSDictionary *pluginConfigurations;
 /**
- @brief Whether or not logging is turned on.
- @discussion Set by @c hideLogs in the configuration file.
+ @brief The build configurations under which logging should be enabled.
+ @discussion Defaults to @c debug. Set by @c loggingBehavior in the configuration file but will inherit the deprecated @c hideLogs flag if @c loggingBehavior is absent.
  */
-@property (nonatomic, assign) BOOL enableLogging;
+@property (nonatomic, assign) CAPInstanceLoggingBehavior loggingBehavior;
 /**
  @brief Whether or not the web view can scroll.
  @discussion Set by @c ios.scrollEnabled in the configuration file. Corresponds to @c isScrollEnabled on WKWebView.
  */
-@property (nonatomic, assign) BOOL enableScrolling;
+@property (nonatomic, assign) BOOL scrollingEnabled;
```

Comment:
This is an unrelated change but was made for the sake of consistency. It not only matches `loggingEnabled`, it better aligns with the config file option and the actual setting on WKWebView so the original name was probably a bad choice.

Reply:
Even if it's for consistency, changes like this should be made in separate PRs, then link with this one and mention that it's for consistency.
If not we can enter into pointless discussions about if we should use `scrollingEnabled` or `scrollEnabled` as the config value name or even another name, instead of focusing on the important changes.